slider/cli.py

```diff
@@ -29,6 +27,10 @@ def maybe_show_progress(it, show_progress, **kwargs):
                 ...
     """
     if show_progress:
+        try:
+            import click
+        except ImportError:
+            raise ImportError("click must be installed to show a progressbar")
```

Comment:
can you do a `raise from` on the original import error to no drop that context for users. There could be a case where the import error is for some weird reason that the user doesn't expect.